### PR TITLE
Init impl

### PR DESCRIPTION
**Pull request by iulianp26**
Read from the video content "related" field, creates a story package container for related items and forwards the new message for further ingestion in neo4j.

See more documentation about Next Video editor processing here: https://sites.google.com/a/ft.com/universal-publishing/documentation/story-specs/3-mar-2017-next-video-platform-work

Tested in video cluster.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  && LDFLAGS="-X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+  && echo "Build flags: $LDFLAGS" \
+  && echo "Fetching dependencies..." \
+  && go get -u github.com/kardianos/govendor \
+  && $GOPATH/bin/govendor sync \
+  && go get -v \
+  && go build -ldflags="${LDFLAGS}" \
+  && mv ${PROJECT} /${PROJECT} \
+  && apk del .build-dependencies \
+  && rm -rf $GOPATH /var/cache/apk/*
```
Comment (tosan88) [inline Dockerfile]: Could you please remove also `/usr/local/go` and  `$GOPATH/src`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+`/__build-info`
+
+Following check is performed for health and gtg endpoints:
+* Checks that the connection to queue can be established.
+
+### Logging
+
+* The application uses [logrus](https://github.com/Sirupsen/logrus); the log file is initialised in [main.go](main.go).
+* Logging requires an `env` app parameter, for all environments other than `local` logs are written to file.
```
Comment (tosan88) [inline README.md]: Is this still valid?

**Comment by tosan88** [comment]
Otherwise lgtm

File: Dockerfile (hunk 1 above)
Comment (iulianp26) [inline Dockerfile]: Done.

File: README.md (hunk 2 above)
Comment (iulianp26) [inline README.md]: Removed logging details.

File: mapper.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+	unmarshalled map[string]interface{}
+}
+
+func (m *relatedContentMapper) mapRelatedContent() ([]byte, string, error) {
+	videoUUID, err := getRequiredStringField(videoUUIDField, m.unmarshalled)
+	if err != nil {
+		return nil, "", err
+	}
+
+	uuid := uuidUtils.NewV5UUIDFrom(videoUUID)
```
Comment (bogdanguranda) [inline mapper.go]: We should use the Deriver when generating an UUID based on another UUID: https://github.com/Financial-Times/uuid-utils-go/blob/master/deriveUUID.go#L27

Let's discuss about this a bit.

Comment (iulianp26) [inline mapper.go]: Done.

**Pull request merged**